backend/inspirehep/records/cli.py

```diff
@@ -179,7 +178,7 @@ def _finished_tasks_count():
     )
     if failures:
         LOGGER.warning(
-            "Got '%d' failures during the recalculation process:", len(failures)
+            f"Got {len(failures)} failures during the recalculation process"
```

Comment:
we had them with ``%d`` for a reason, is the aggregation working well with this framework?